plugin/OmniSharp.vim

```diff
@@ -76,6 +76,15 @@ if g:Omnisharp_stop_server == 1
   autocmd VimLeavePre * call OmniSharp#AskStopServerIfRunning()
 endif
 
+" Initialize OmniSharp as an asyncomplete source
+if exists('*asyncomplete#register_source')
```

Comment:
Would this work?

```vim
au User asyncomplete_setup call asyncomplete#register_source({ ... })
```

Reply:
Yes that works great, thanks! I had tried the autocmd earlier without success, but I think I had some other issues at the time that were causing problems.